lib/src/modules/Button/ButtonBase.js

```diff
@@ -18,6 +18,8 @@ function ButtonBase({
   buttonRef,
   ...rest
 }) {
+  const { defaultFillColor, ...restProps } = rest;
```

Comment:
same as the one above! why not just add `defaultFillColor` after line 18?

Reply:
fair point on this one - I'll get it changed